admin/admin-filters-columns.php

```diff
@@ -323,7 +323,7 @@ public function ajax_update_term_rows() {
 
 		$translations = empty( $_POST['translations'] ) ? array() : explode( ',', $_POST['translations'] ); // collect old translations
 		$translations = array_merge( $translations, $this->model->term->get_translations( (int) $_POST['term_id'] ) ); // add current translations
-		$translations = array_unique( $translations ); // remove doublons
+		$translations = array_unique( $translations ); // remove doubloons
```

Comment:
doublon is a French word meaning "duplicate". So I would rather use "duplicates" rather than "doubloons" (which according to the dictionnary is an old Spanish coin, out of context here).